sass/tokens/register/_index.scss

```diff
@@ -1,4 +1,4 @@
-/// # Registered Functions
+/// # Pre-Registered Functions
```

Comment:
@mirisuzanne Should the 'math' functions in `tokens/register` show in the docs? I think this is the only module with a nested folder. Is that what's causing 'Pre-Registered Functions' to not show in the docs?